system/web/Renderer.cfc

```diff
@@ -694,6 +694,29 @@ component accessors="true" serializable="false" extends="coldbox.system.Framewor
 		return locateView(arguments.view);
 	}
 
+	public string function elixir( required string fileName, string buildDirectory = 'build' ) {
```

Comment:
1. Please move to the HTML Helper
2. Add documentation for function and arguments